### PR TITLE
Change YouTube subscription import tutorial

### LINKED ISSUE #194: [FAQ] Update import subscriptions tutorial
NewPipe uses Google takeouts for subscription imports for YouTube since 0.20.3 (see https://github.com/TeamNewPipe/NewPipe/pull/4759).
Apart from that, the screenshots are outdated.

### PR DESCRIPTION
**Pull request by TobiGr**
Closes #194

### CODE REVIEW

File: _tutorials/import-export-data.html [hunk 1]
```diff
@@ -32,12 +32,15 @@ <h3 id="import-youtube">Import YouTube subscriptions</h3>
...
-                    <li>Go to the <a href="https://www.youtube.com/subscription_manager?action_takeout=1">YouTube subscriptions manager</a>.</li>
+                <ol>
+                    <li>Go to the <a href="https://takeout.google.com/takeout/custom/youtube">Google takeout manager</a>.</li>
                     <li>Log in if asked.</li>
-                    <li>A download will start. It contains your exported YouTube subscriptions.</li>
-                    <li>Go back to NewPipe and tap on the import file at the bottom of the screen. Now, select your downloaded YouTube export file.</li>
-                </ul>
+                    <li>Click on "All data included", then on "Deselect all", then select only "subscriptions" and click "OK".</li>
+                    <li>Click on "Next step" and then on "Create export".</li>
+                    <li>Click on the "Download" button after it appears and</li>
```
Comment (Stypox) [inline _tutorials/import-export-data.html:40]:
```suggestion
                    <li>Click on the "Download" button after it appears.</li>
```

File: _tutorials/import-export-data.html [hunk 2]
```diff
@@ -32,12 +32,15 @@ <h3 id="import-youtube">Import YouTube subscriptions</h3>
...
+                    <li>From the downloaded takeout zip extract the .json file. It is usually located under "YouTube and YouTube Music/subscriptions/subscriptions.json".</li>
```
Comment (Stypox) [inline _tutorials/import-export-data.html:41]: Would it be possible to add the <code> tag around `zip`, `.json` and `YouTube and YouTube Music/subscriptions/subscriptions.json`?

**Review by Stypox — CHANGES_REQUESTED** [review]

File: _tutorials/import-export-data.html (hunk 2 above)
Comment (B0pol) [reply _tutorials/import-export-data.html:41]: the `<code>` tag

**Review by Stypox — CHANGES_REQUESTED** [review]
After this, feel free to merge ;-)

File: _tutorials/import-export-data.html [hunk 3]
```diff
@@ -32,12 +32,15 @@ <h3 id="import-youtube">Import YouTube subscriptions</h3>
...
+                    <li>Go to the <a href="https://takeout.google.com/takeout/custom/youtube">Google takeout manager</a>.</li>
                     <li>Log in if asked.</li>
-                    <li>A download will start. It contains your exported YouTube subscriptions.</li>
-                    <li>Go back to NewPipe and tap on the import file at the bottom of the screen. Now, select your downloaded YouTube export file.</li>
-                </ul>
+                    <li>Click on "All data included", then on "Deselect all", then select only "subscriptions" and click "OK".</li>
+                    <li>Click on "Next step" and then on "Create export".</li>
+                    <li>Click on the "Download" button after it appears.</li>
+                    <li>From the downloaded takeout <code>zip</code> extract the <code>.json</code> file. It is usually located under <pre style="white-space: normal">YouTube and YouTube Music/subscriptions/subscriptions.json</pre>.</li>
+                    <li>Go back to NewPipe, tap on "Import from > YouTube", hit "import file" at the bottom of the screen and select the .json file.</li>
```
Comment (Stypox) [inline _tutorials/import-export-data.html:42]: Oh, sorry, I seem to have mis-sent the previous review. I meant that this should be changed and then everything is good.
```suggestion
                    <li>Go back to NewPipe, tap on "Import from > YouTube", hit "import file" at the bottom of the screen and select the <code>.json</code> file.</li>
```

**Review by Stypox — CHANGES_REQUESTED** [review]

File: _tutorials/import-export-data.html (hunk 3 above)
Comment (TobiGr) [reply _tutorials/import-export-data.html:42]: Done in 84d7cdb2d12e9a6de834eae6539dcef31533fb75

**Pull request merged**